code_soup/common/utils/checkpoints.py

```diff
@@ -0,0 +1,12 @@
+import torch
+
+
+class Checkpoints:
+    @classmethod
+    def save(self, PATH, model):
+        torch.save(model.state_dict(), PATH)
+
```

Comment:
This implementation is model specific, we may need to save the optimiser too, Checkpoints save both the model state and the checkpoint state